website/docs/reference/developer-readme.md

```diff
@@ -7,21 +7,45 @@ description: SynapseML Development Setup
 
 # SynapseML Development Setup
 
+## Windows
+1) Install JDK 11
+    - https://docs.microsoft.com/en-us/java/openjdk/download#openjdk-11
+1) Configure Hadoop "winutils"
+    - Download from https://github.com/cdarlint/winutils/tree/master/hadoop-3.2.2/bin
+    - Create new folder, "C:\Program Files\Hadoop\bin", and place winutils.exe and hadoop.dll inside
+    - Add HADOOP_HOME="C:\Program Files\Hadoop" to Env Vars
```

Comment:
Is it a good idea to configure Hadoop along with winutils.exe and hadoop.dll: https://hadoop.apache.org/releases.html

Reply:
Previously, hadoop DDL was not required... but with the latest release Mark and I were having trouble getting this to work without it.

Reply:
True, Hadoop dll is required. but my thought was to see if it is a better idea to set up Hadoop itself instead of just winutils and dll file.